presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -206,6 +207,31 @@ public void testArrayContains()
     }
 
     @Test
+    public void testArrayJoin()
+            throws Exception
+    {
+        assertFunction("ARRAY_JOIN(ARRAY [1, 2, 3], ';', 'N/A')", VARCHAR, "1;2;3");
+        assertFunction("ARRAY_JOIN(ARRAY [1, 2, null], ';', 'N/A')", VARCHAR, "1;2;N/A");
+        assertFunction("ARRAY_JOIN(ARRAY [1, 2, 3], null)", VARCHAR, null);
```

Comment:
Since you're passing NULL as the second argument the invocation of the function is skipped, so this test probably isn't that useful. I would use a non-NULL value for this and the next two tests